src/functions/arraySort.js

```diff
@@ -9,7 +9,7 @@ module.exports = async (d) => {
   }
 
   d.arrays[name] = d.arrays[name].sort((a, b) =>
-    type === "asc" ? a - b : b - a,
+    type === "asc" ? (a < b ? -1 : a > b ? 1 : 0) : (a < b ? 1 : a > b ? -1 : 0),
   );
```

Comment:
Maybe doing it like this:

```js
.sort((a, b) =>
    typeof a === "number" && typeof b === "number" ? a - b :
    typeof a === "string" && typeof b === "string" ? a.localeCompare(b) :
    typeof a === "number" ? -1 : 1
);
```

is more readable and robust? you could use .reverse() at the end to reverse the array if type is desc